Rakefile

```diff
@@ -71,19 +65,19 @@ namespace :postprocess do
     end
 
     # Remove links that are the same page as their parent, remove them.
-    doc.css("ol.toc li a").each do |item|
-      item_href = item.attributes["href"].to_s.gsub(/#.*\Z/, "")
-      branch = item.parent.parent.parent
+    #doc.css("ol.toc li a").each do |item|
+    #  item_href = item.attributes["href"].to_s.gsub(/#.*\Z/, "")
+    #  branch = item.parent.parent.parent
 
-      if branch.node_name == "li"
-        a = branch.at("a")
-        a_href = a.attributes["href"].to_s.gsub(/#.*\Z/, "")
+    #  if branch.node_name == "li"
+    #    a = branch.at("a")
+    #    a_href = a.attributes["href"].to_s.gsub(/#.*\Z/, "")
 
-        if a_href == item_href
-          item.remove
-        end
-      end
-    end
+    #    if a_href == item_href
+    #      item.remove
+    #    end
+    #  end
+    #end
   end
```

Comment:
Currently it does this to the table of contents.

<img width="337" alt="screen shot 2015-12-31 at 4 14 09 pm" src="https://cloud.githubusercontent.com/assets/430277/12066086/99ad2640-afd9-11e5-8a25-6feda14f7412.png">

Need to investigate more.